packages/cyberstorm/package.json

```diff
@@ -15,9 +15,12 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.17.0",
-    "@fortawesome/fontawesome-svg-core": "^6.2.0",
-    "@fortawesome/free-solid-svg-icons": "^6.2.0",
+    "@fortawesome/fontawesome-pro": "^6.2.1",
```

Comment:
As discussed in some other PR, if these icon packages are updated, they should be updated everywhere in the monorepo, since this package seems to have rather unstable TS across minor versions. In this case, I don't see updates to sertra's package.json?

Reply:
https://github.com/thunderstore-io/thunderstore-ui/pull/591#discussion_r1054030900 Going to be handled within this checking I'm doing.